lib/web_ui/dev/browser_roller.dart

```diff
@@ -0,0 +1,318 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:args/args.dart';
+import 'package:http/http.dart';
+import 'package:path/path.dart' as path;
+
+import 'browser_lock.dart';
+import 'common.dart';
+import 'utils.dart';
+
+final ArgParser _argParser = ArgParser(allowTrailingOptions: false)
+  ..addOption(
+    'version-tag',
+    abbr: 't',
+    help: 'The tag to assign to the uploaded package. This label can be used '
+          'to refer to this version from .ci.yaml.',
+    mandatory: true,
+    valueHelp: 'VERSION'
+  )..addOption(
+    'platform',
```

Comment:
There's not a strong case for this variable.

Reply:
Removed